doc/tools/apigen.py

```diff
@@ -282,24 +282,23 @@ def generate_api_doc(self, uri):
         ad += title + '\n' + self.rst_section_levels[1] * len(title) + '\n\n'
 
         ad += '.. automodule:: ' + uri + '\n\n'
-        ad += '.. currentmodule:: ' + uri + '\n\n'
+        # ad += '.. currentmodule:: ' + uri + '\n\n'
```

Comment:
> Unsure about the changes in the ad +=  lines, but merging to get CI green.

Autosummary now warns if the `.. currentmodule` is specified and then again used as a prefix for the items in the `.. autosummary` directive.

Which makes me realize that this commented line was an oversight leftover from debugging and should have been uncommented. It doesn't seem to cause an problems but let's put it back in to be safe.

I'm a bit fuzzy on this because I can't find a good answer in Sphinx docs to how those two directives interact and why it seems to work with and without.